classes/PodsAdmin.php

```diff
@@ -573,6 +573,14 @@ public function admin_menu() {
 
 			if ( empty( $all_pods ) ) {
 				unset( $admin_menus['pods'] );
+
+				if ( 'pods' === pods_v( 'page', 'get' ) ) {
+					// Replace `pods` page param with first existing pod page and redirect.
+					$url = add_query_arg( 'page', key( $admin_menus ) );
+					$url = get_site_url( null, $url );
+					wp_redirect( $url );
```

Comment:
:warning: wp_redirect() found. Using wp_safe_redirect(), along with the allowed_redirect_hosts filter if needed, can help avoid any chances of malicious redirects within code. It is also important to remember to call exit() after a redirect so that no other unwanted code is executed.

<sub>hash: 437ac422441dc926853f890489cd6508</sub>